modules/test.js

```diff
@@ -0,0 +1,48 @@
+import { saveLocal, collectionMethod } from './localStorage_Functions.js';
+
+export const addBtn = document.querySelector('.add-btn');
+/*eslint-disable*/
```

Comment:
Hi, You have done an amazing job, and your App is working fine. I would advise you not to disable ESLint.
**ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs**.
Therefore you can fix the bugs manually, and make sure your App works fine.